src/bridge/cockpitcpusamples.c

```diff
@@ -112,3 +112,81 @@ cockpit_cpu_samples (CockpitSamples *samples)
   g_strfreev (lines);
   g_free (contents);
 }
+
+gchar*
```

Comment:
All new helper functions (all new ones except `cockpit_cpu_temperature`)in this file should be `static`, these are not API but just internal implementation details.

Reply:
Done.